pai-management/k8sPaiLibrary/maintainlib/kubectl_install.py

```diff
@@ -71,11 +72,16 @@ def kubectl_configuration_generate(self):
 
     def kubectl_ready_test(self):
 
+        times = 0
+
         while True:
             res = common.execute_shell_return("kubectl get node", "There will be a delay after installing, please wait.")
-
+            times = times + 1
             if res == True:
                 break
+            if times == 10:
+                self.logger.error("kubectl ready test failed. Exit paictl.")
+                sys.exit(1)
 
```

Comment:
What would happen after the retry count down?
How long will it wait?

Reply:
A short time, I think it's ok.

Reply:
Would it quit the deployment if fails?
Should it be ready in a short time?

Reply:
I'm worry about it would fail because of time out.
It's fine if we are sure it would ready in a short time.

Reply:
The command failed is ok. Because sometimes, the api-server of the master node may be not ready. And if every thing is fine, it will started soon. And then the kubectl could work well. 
But if this command fail 10 times, I will assume something wrong happened.